state/txindex/indexer_service.go

```diff
@@ -55,37 +57,45 @@ func (is *IndexerService) OnStart() error {
 		return err
 	}
 
+	is.wg = &sync.WaitGroup{}
```

Comment:
Adding a goroutine here does not appear related to the main purpose of this PR. Let's please leave this out, if there's a good reason for it we can do it in a separate PR.

Reply:
removed